clients/drcachesim/tracer/output.cpp

```diff
@@ -728,9 +728,9 @@ output_buffer(void *drcontext, per_thread_t *data, byte *buf_base, byte *buf_ptr
               size_t header_size)
```

Comment:
There is a test failure: is it a known one?  Does it repeat?

Reply:
It was `tool.drcachesim.opcode_mix`

```
Encoding size 2 != instr size 6
drcachesim: /home/runner/work/dynamorio/dynamorio/clients/drcachesim/reader/reader.cpp:203: virtual bool dynamorio::drmemtrace::reader_t::process_input_entry(): Assertion `false' failed.
```

I don't think we've seen this before.

Reply:
It didn't repeat though when I reran the workflow.

Reply:
Created #6314